### DISCUSSION TITLE
how to choose installation folder?

### DISCUSSION THREAD

**Discussion by Laniatus** [LykosAI/StabilityMatrix#90] [Q&A]
my main storage is a low space ssd. to install on another hd that I have on the computer that has more space. How do I change the installation directory?

**Comment by brknsoul** [comment]
StabilityMatrix Setting cog (at the bottom left), Select New Data Directory.

![image](https://github.com/LykosAI/StabilityMatrix/assets/1969381/8ad4426a-8b25-4ef2-9b31-61ee3b6d5317)

**Reply by drdancm** [reply]
I have run into some problems on initial install and then with an update.
1 Simply create a directory on your HD which has lots more space than SSD. Then after downloading StabMatrix, unzip it, and place the exe file in your newly created directory. That seems to work well. I think one time I ran the install file from another locations and it messed up.

2 When the install asks about location, browse to your new folder on your HD and also tell it that you want to keep it all in one place (I don't remember the exact wording). This will make your single new folder contain everything Data and Packages, so if you wish you can just copy it to another drive, or computer and not have to reload from scratch. Very handy.

**Comment by monkyboom** [comment]
it dosnt find anything i have installed.
